test/unit/config/config_manage/1607_root_filters/config/galaxy.ini

```diff
@@ -144,15 +144,6 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # config/tool_conf.xml does not exist).
 #tool_config_file = config/tool_conf.xml,config/shed_tool_conf.xml
 
-# Enable / disable checking if any tools defined in the above non-shed
-# tool_config_files (i.e., tool_conf.xml) have been migrated from the Galaxy
-# code distribution to the Tool Shed.  This setting should generally be set to
-# False only for development Galaxy environments that are often rebuilt from
-# scratch where migrated tools do not need to be available in the Galaxy tool
-# panel.  If the following setting remains commented, the default setting will
-# be True.
-#check_migrate_tools = True
-
```

Comment:
I don't think this and the following file should be modified?

Reply:
I don't mind. But why would we need this option to be part of a test file if it's removed? (I don't mind either way, just double-checking).

Reply:
I guess the point is that we have a config as it used to be, so we can check the config_manage script. It does make sense to keep it imo.

Reply:
Done!